README.md

```diff
@@ -1,10 +1,35 @@
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/11a3ucotxcl9746k/branch/master?svg=true)](https://ci.appveyor.com/project/BHoMBot/ClimateEmergency-Toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/ClimateEmergency_Toolkit/ClimateEmergency_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build?definitionId=198&_a=summary)
```

Comment:
DefinitionID for Azure pipeline should be 52 - full badge is `[![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/ClimateEmergency_Toolkit/ClimateEmergency_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=52&branchName=master)`